src/vision/pi_vision/pi_face_tracker/nodes/face_tracker.py

```diff
@@ -258,7 +263,10 @@ def __repr__(self):
         )
 
     def update_bounding_box(self):
-        self.bounding_size = self.pt2[1] - self.pt1[1]
+        #is dp 0.22 the correct size for width or height of face?
+        #now using width instead of height as initialized
```

Comment:
Of course, it might be the case that CMT is not actually placing the bounding box on the actual head, chin to crown, but on something smaller or larger?  If the CMT bounding-box size is different from the size of the human head, then there would need to be a correction for that.